ui/src/components/Actuator.vue

```diff
@@ -0,0 +1,139 @@
+<script setup lang="ts">
+import { getActuator, getMetricsAll } from "@/api/api";
+import { ref, watch } from "vue";
+import ActuatorItem from "./ActuatorItem.vue";
+import SearchBar from "@/components/SearchBar.vue";
+
+const actuator = ref({});
+const metrics = ref({});
+
+const loadActuator = async () => {
+  let result = (await getActuator())["_links"];
+  let list = [];
+  for (let key in result) {
+    const item = result[key];
+    item["key"] = key;
+    list.push(result[key]);
+  }
+  actuator.value = list;
+  console.log(JSON.stringify(actuator.value, null, 2));
+};
+const loadMetrics = async () => {
+  metrics.value = await getMetricsAll();
+
+  // preload search values
+  filteredLines();
+};
+
+loadMetrics();
+loadActuator();
+function downloadJSON(json, filename: string) {
+  const cleanedUp = removeFields(json);
+  var dataStr =
```

Comment:
Please use `const` and `let` instead of `var`

Reply:
Done